src/macaron/__main__.py

```diff
@@ -118,6 +119,31 @@ def verify_policy(verify_policy_args: argparse.Namespace) -> int:
     return os.EX_USAGE
 
 
+def verify_repo_finder() -> int:
```

Comment:
Is this verify functionary available for the user or do we only have it for the integration tests only? If it's intended for the user, I think we need to update the Sphinx documentation too (I'm happy to have it done in another PR if that's the case).

Reply:
I don't think there is much value in making it available to the user. It has a very specific use currently, and extending it to be more useful for users might not be a good use of time (in my opinion).

Reply:
Soungs good to me. 

The only concern is that with this design, this `verify-repo-finder` command will be shown when the user run `macaron --help`.
```
usage: macaron [-h] [-V] [-v] [-o OUTPUT_DIR] [-dp DEFAULTS_PATH] [-lr LOCAL_REPOS_PATH]
               {analyze,dump-defaults,verify-policy,verify-repo-finder} ...

positional arguments:
  {analyze,dump-defaults,verify-policy,verify-repo-finder}
                        Run macaron <action> --help for help

...
```
But I guess it won't pose any problem (at least for me). Let's keep it this way for now.

Reply:
Since we have `verify-policy` as well, to avoid confusion we can change `verify-repo-finder` to `test-repo-finder` to indicate this is just for testing purposes. What do you think?

Reply:
That works for me.

Reply:
Sounds good.

Reply:
Hi Ben. What do you think about moving your `test_repo_finder()` function from `src/macaron/__main__.py` into a new Python script e.g. in `tests/e2e/repo_finder.py`, and instead of running `macaron test-repo-finder`, we can run the `python tests/e2e/repo_finder.py` script in our integration tests?

The `tests/e2e/repo_finder.py` script can be run in `integration_test.sh` similar to how the `tests/e2e/compare_e2e_result.py` is being run.

I prefer this to adding the `test-repo-finder` command to Macaron, as it may cause confusion for a user who relies on the CLI help. At the same time, `test_repo_finder()` is only for testing and not actually being used in Macaron, and hence I don't think it should stay in `src`.

Reply:
I think that's a good idea.